substratevm/DebugInfo.md

```diff
@@ -61,14 +63,18 @@ native-image -H:GenerateDebugInfo=1 \
     org.my.Hello
 ```
 By default, the cache of application, GraalVM, and JDK sources is
-located under local directory sources. The `DebugInfoSourceCacheRoot`
-option can be used to specify an alternative location for the top level
-directory. As an example, the following variant of the previous
-command specifies the same target but employs an absolute path:
+created in a directory named `sources`. The `DebugInfoSourceCacheRoot`
+option can be used to specify an alternative path, which can be
+absolute or relative. In the latter case the path is interpreted
+relative to the target directory for the generated native image
+specified via option `-H:Path` (which defaults to the current
+working directory). As an example, the following variant of the
+previous command specifies an absolute temporary directory path
+constructed using the current process id:
 ```shell
 SOURCE_CACHE_ROOT=$PWD/sources
 native-image -H:GenerateDebugInfo=1 \
```

Comment:
This should be `SOURCE_CACHE_ROOT=/tmp/$$/sources`

Reply:
I also suggested adding a followup line saying

> The resulting cache directory will be something like /tmp/1272696/sources.

That's not necessary but I thought it would be useful. Your choice.

Reply:
Fixed and fixed.